main.sh

```diff
@@ -114,4 +114,4 @@ data="$data&openSslVersion=$(serverdiagnostic openssl 1)"
 ################################
 # Execute the request
 ################################
-curl -k -H "Authorization: Bearer $SD_MAGEDIAGNOSTICS_SERVER_KEY" --data "$data" $SD_MAGEDIAGNOSTICS_API_ENDPOINT
+curl -k -H -s "Authorization: Bearer $SD_MAGEDIAGNOSTICS_SERVER_KEY" --data "$data" $SD_MAGEDIAGNOSTICS_API_ENDPOINT
```

Comment:
I don't think this is right. I'm pretty sure `-H` has to be followed by the header. I think this can be

```
curl -ks -H 'Authorization: ...'
```

did you test this?